File: TestWCF/TestWCF2/Calculator.svc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.Text;

namespace TestWCF2
{
    // 注意: 如果更改此处的类名 "Calculator"，也必须更新 Web.config 中对 "Calculator" 的引用。
    public class Calculator : ICalculator
    {
        public double Add(double n1, double n2)
        {
            return n1 + n2;
        }
        public double Subtract(double n1, double n2)
        {
            return n1 - n2;
        }
        public double Multiply(double n1, double n2)
        {
            return n1 * n2;
        }
        public double Divide(double n1, double n2)
        {
            return n1 / n2;
        }
    }
}

File: TestWCF/TestWCF/Calculator.svc.vb
' 注意: 如果更改此处的类名 "Calculator"，也必须更新 Web.config 和关联的 .svc 文件中对 "Calculator" 的引用。
Public Class Calculator
    Implements ICalculator

    Public Sub DoWork() Implements ICalculator.DoWork
    End Sub

End Class
